### PR TITLE
Make editor panel resizable

### LINKED ISSUE #193: Make editor panel resizable
## Need
```gherkin
As a user
I want to be able to resize the editor panel
So that I can use the screen space efficiently.
```
## Deliverables
- [ ] The separator between the editor and preview panes can be dragged to resize them (when the editor pane enlarges, the preview pane shrinks and vice versa)

#### Prerequisites
- Use the React SplitPane component located [here](https://github.com/ubervu/react-split-pane)

#### TODO
- [ ] Add the editor and preview pane components to a SplitPane 
- [ ] Adapt old tests and add new ones

### PR DESCRIPTION
**Pull request by bogdan-stefan-hs**
Closes #193.

### CODE REVIEW

**Comment by bogdan-stefan-hs** [comment]
@skidding 
Please take a look at the code.

Tested on React 0.13 and 15.

The LESS is a bit of a mess with some selectors having '>' and some not, but after we merge everything from the uberVU fork we should separate it into more files (as well as the jsx), so i guess we shouldn't worry too much about it now.

resize

File: packages/react-component-playground/test/components/component-playground/selected-fixture-and-editor/events/handlers.js [hunk 1]
```diff
@@ -60,4 +61,14 @@ describe(`ComponentPlayground (${FIXTURE}) Events Handlers`, function() {
             .to.equal(JSON.stringify(fixture.state.fixtureContents, null, 2));
     });
   });
+
+  it('should set splitpane onChange to localstorage lib set', function() {
```
Comment (skidding) [inline packages/react-component-playground/test/components/component-playground/selected-fixture-and-editor/events/handlers.js]: How about more humane test cases? E.g. "should save splitpane position in local storage on change"

File: packages/react-component-playground/test/components/component-playground/selected-fixture-and-editor/render/dom.js [hunk 2]
```diff
@@ -45,4 +40,18 @@ describe(`ComponentPlayground (${FIXTURE}) Render DOM`, function() {
     expect($(component.refs.editor)
            .hasClass(style['invalid-syntax'])).to.be.true;
   });
+
+  it('should render a splitpane ', function() {
+    expect(component.refs.splitPane).to.exist;
+  });
+
+  it('should have proper split orientation on SplitPane', function() {
```
Comment (skidding) [inline packages/react-component-playground/test/components/component-playground/selected-fixture-and-editor/render/dom.js]: `splitpane` or `SplitPane`? Decide :)

**Comment by skidding** [comment]
Finished review. Beautiful! 🌷 

Final note: Did you get to test on other browsers besides Chrome? If would be valuable to know if it runs under (any) IE, because it's useful to load your components cross browser. Thanks.

**Comment by bogdan-stefan-hs** [comment]
@skidding
I updated the tests.

Didn't get the chance to test under IE, but I'll do so when I get access to a Windows machine and I'll keep you posted.

File: packages/react-component-playground/test/components/component-playground/selected-fixture-and-editor/events/handlers.js [hunk 3]
```diff
@@ -62,13 +62,11 @@ describe(`ComponentPlayground (${FIXTURE}) Events Handlers`, function() {
...
   });
 
-  it('should set splitpane onChange to localstorage lib set', function() {
-    expect(component.refs.splitPane.props.onChange())
-      .to.equal(localStorageLib.set('splitPos'));
-  });
+  it('should have split-pane position in local storage on change', function() {
+    var size = 10;
 
-  it('should set splitpane defaultSize to localstorage lib get', function() {
```
Comment (skidding) [inline packages/react-component-playground/test/components/component-playground/selected-fixture-and-editor/events/handlers.js]: @bogdan-stefan-hs why did you remove this test? Is it not relevant to test that the defaultSizes is ready from the local storage?

I was just referring to humanising the test case strings in my previous comment.

Comment (bogdan-stefan-hs) [inline packages/react-component-playground/test/components/component-playground/selected-fixture-and-editor/events/handlers.js]: Oh, I thought you meant testing the props isn't the way to go, so I added a new test for the onChange effects.

The two tests are only asserting that props are sent correctly. The new one asserts that `splitPane` actually calls that callback, but I guess this is `react-split-pane`'s job, so we don't need to test that.

I'm going back to the two initial tests now.

Comment (skidding) [inline packages/react-component-playground/test/components/component-playground/selected-fixture-and-editor/events/handlers.js]: Cool. Sorry for the confusion.
